content/concepts/introduction/intro-to-consensus.md

```diff
@@ -0,0 +1,63 @@
+- Metadata about the current ledger version, such as its ledger index, a [cryptographic hash](https://en.wikipedia.org/wiki/Cryptographic_hash_function) that uniquely identifies its contents, and information on the parent ledger was used as a basis for building this one.
+
+[![Figure 1: Anatomy of a ledger version, which includes transactions, state, and metadata](img/anatomy-of-a-ledger-simplified.png)](img/anatomy-of-a-ledger-simplified.png)
+<!--{# Diagram source: https://docs.google.com/presentation/d/1mg2jZQwgfLCIhOU8Mr5aOiYpIgbIgk3ymBoDb2hh7_s/ #}-->
+
+Each ledger version is numbered with a _ledger index_ and builds on a previous ledger version whose index is one less, going all the way back to a starting point called the _genesis ledger_ with ledger index 1. Like Bitcoin and other blockchain technologies, this forms a public history of all transactions and their results. Unlike most blockchain technologies, each new "block" in the XRP Ledger contains the entirety of the current state, so you don't need to collect the entire history to know what's happening now.[¹](#footnote-1)
+
+The main job of the XRP Ledger Consensus Protocol is to agree on set of transactions to apply to the previous ledger, apply them in a well-defined order, then confirm that everyone got the same results. When this happens successfully, a ledger version is considered _validated_, and final. From there, the process continues by building the next ledger version.
+
+
+## Trust-Based Validation
+
+The core principle behind the XRP Ledger's consensus mechanism is that a little trust goes a long way. Each participant in the network chooses a set of _validators_, servers [specifically configured to participate actively in consensus](run-a-rippled-validator.html), run by different parties who are expected to behave honestly most of the time. More importantly, the set of chosen validators should not be likely to collude with one another to break the rules in the exact same way.
+
+As the network progresses, each server listens to its trusted validators[²](#footnote-2); as long as a large enough percentage of them agree that a set of transactions should occur and that a given ledger is the result, the server declares a consensus. If they don't agree, validators modify their proposals to more closely match the other validators they trust, repeating the process in several rounds until they reach a consensus.
+
+[![Figure 2: Consensus rounds. Validators revise their proposals to match other validators they trust](img/consensus-rounds.png)](img/consensus-rounds.png)
+
+It's OK if a small proportion of validators don't operate properly all the time. As long as fewer than 20% of trusted validators are faulty, consensus can continue unimpeded; and confirming an invalid transaction would require over 80% of trusted validators to collude. (Anywhere in between, and the network simply stops making progress.)
```

Comment:
I'm curious about the "Anywhere in between" text in the statement in parentheses - can you help this newbie brain work through the thought? 🙏 😂 Something like - If anywhere between X% and X% of validators <do what>, the network simply stops making progress ?

Reply:
If more than 20% of validators on your UNL behave badly (aka. "byzantine" if you want to google it or read papers about these protocols), then your server will stop creating new ledgers. Since likely the overlap between UNLs is rather high, this could also mean that the whole network stops.

If more than 80% of validators on your UNL do something you do not agree with, your server will NOT necessarity stop. It'll either accept the change that you disagree with (if it still makes sense from the point of view of the software - e.g. the AccountReserve fee being increased to 100 XRP while you voted for 8 XRP) and move forward or it will stop completely if the disagreement isn't just something that you personally don't want to happen, but also something that your software won't allow - e.g. a transaction with an invalid signature.